    Add mvsata support to SHEEVAPLUG

Updated patch from drEagle <drEagle@DOUKKI.NET>:

https://lists.debian.org/debian-arm/2014/07/msg00043.html

Index: u-boot/include/configs/sheevaplug.h
===================================================================
--- u-boot.orig/include/configs/sheevaplug.h
+++ u-boot/include/configs/sheevaplug.h
@@ -30,6 +30,7 @@
 #include <config_cmd_default.h>
 #define CONFIG_CMD_DHCP
 #define CONFIG_CMD_ENV
+#define CONFIG_CMD_IDE
 #define CONFIG_CMD_MII
 #define CONFIG_CMD_MMC
 #define CONFIG_CMD_NAND
@@ -106,4 +107,15 @@
 #define CONFIG_CMD_MTDPARTS
 #define CONFIG_LZO
 
+/*
+ * SATA driver configuration
+ */
+#ifdef CONFIG_CMD_IDE
+#define __io
+#define CONFIG_IDE_PREINIT
+#define CONFIG_DOS_PARTITION
+#define CONFIG_MVSATA_IDE_USE_PORT0
+#define CONFIG_SYS_ATA_IDE0_OFFSET     MV_SATA_PORT0_OFFSET
+#endif /* CONFIG_CMD_IDE */
+
 #endif /* _CONFIG_SHEEVAPLUG_H */
